src/primitives.ts

```diff
@@ -35,6 +35,7 @@ export enum EventName {
   PAGE_CLICKED = 'Page Clicked',
   PAGE_VIEWED = 'Page Viewed',
   SEARCH_BAR_CLICKED = 'Search Bar Clicked',
+  SENTIMENT_CLICKED = 'Sentiment Clicked',
```

Comment:
This keeps going on the "X clicked" pattern which I will end up with a lot of "primitive" events vs using properties and specifics vs cluttering this top-level namespace. I know I'm delayed on that README, but I think that's an important distinction, and I will check in with others on that for event design before locking that in!

In this specific case it appears this is less about a click and more about a submit, so I see two possible options:

1. Go with a `SENTIMENT_SUBMITTED` event that we keep in general (IMO preferred as maybe we unlink this from being a click at some point, or there could be multiple clicks in the process, and other products reasonably may use sentiment)

2. Go even more general and create a new simple `CLICK_EVENT` to start setting up the easier hierarchy for primitives.

I want to get wider agreement for the second one, so I'm thinking that we shouldn't block this implementation on that and go for the first one but open to discussion here as the rules here still arent locked!

Reply:
First option sounds good!